packages/angular_devkit/build_angular/test/browser/rebuild_spec_large.ts

```diff
@@ -347,4 +347,42 @@ describe('Browser Builder rebuilds', () => {
     ).toPromise();
     await run.stop();
   });
+
+  it('forkTypeChecker emits interface error on rebuilds', async () => {
+    host.writeMultipleFiles({
+      'src/interface.ts': `export interface Foo { bar: boolean };`,
```

Comment:
This isn't quite what https://github.com/angular/angular-cli/issues/13209 reports. 

In that issue the interface is never exported. Since there are no imports or exports, the file is not an ES Module. TS then interprets it as a global definition, like .d.ts files.

I think that is at the core of this problem.